### PR TITLE
Improve README - Add PT-BR Translation

### LINKED ISSUE #10: Improve README.md - Adding PT-BR translation
_Originally posted by @myakiju in https://github.com/naXa777/spring-cloud-example/issues/1#issuecomment-429409771_

I thought it would be good to have separated issues for each language. So you can keep track on what's finished and what is not.

Do you agree, @naXa777 ? 

Regards

### PR DESCRIPTION
**Pull request by o-niko**
First version of PT-BR translation for the README file

Closes #10

### CODE REVIEW

File: readme-translations/README.br.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
+# Spring Cloud Example
+
+[![Build Status](https://travis-ci.com/naXa777/spring-cloud-example.svg?branch=master&style=flat)](https://travis-ci.com/naXa777/spring-cloud-example)
+
+Esse playground é para aprender [Spring Cloud](https://cloud.spring.io/) e [Kotlin](https://kotlinlang.org/) através de exemplos.
+
+Leia em outros idiomas: 
+* [На русском](../README.md) - Russo
+* [Bahasa Indonesia](README.id.md) - Indonésio
+* [In het Nederlands](README.nl.md) - Holândes
```
Comment (naXa777) [inline readme-translations/README.br.md]: Do not forget to add link to English readme

Comment (myakiju) [reply readme-translations/README.br.md]: done :)

Comment (naXa777) [reply readme-translations/README.br.md]: nice :) but what about crosslinks to this file from other readmes?

File: readme-translations/README.br.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+#### Painel de Controle - Dashboard
+
+O Eureka possui um painel de controle web habilitado por padrão. Ele exibe informação úteis a respeito do servidor. 
+ 
+Em ambiente local, o painel de controle está disponível no caminho http://localhost:8761
+
+### Serviço de Clima
+
+| Module          | URL                        |
```
Comment (naXa777) [inline readme-translations/README.br.md]: Módulo

File: readme-translations/README.br.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+| :-------------: | :------------------------: |
+| weather-service | `http://host:port/weather` |
+
+A anotação `@EnableDiscoveryClient` é usada para transformar o `WeatherServiceApplication` num cliente do Discovery Server e faz com que ele se registre no Discovery Server em sua inicialização. 
+
+### Weather App
+
+| Module      | URL                                |
```
Comment (naXa777) [inline readme-translations/README.br.md]: Módulo

File: readme-translations/README.br.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+| :---------: | :--------------------------------: |
+| weather-app | `http://host:port/current/weather` |
+
+### Serviço de Data e Hora
+
+| Module           | URL                         |
```
Comment (naXa777) [inline readme-translations/README.br.md]: `Módulo` here and below

File: readme-translations/README.br.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+Em Sistemas Distribuídos existe apenas uma certeza... FALHAS SÃO INEVITÁVEIS.
+
+Um efeito particularmente ruim em falhas de sistemas distribuídos é a [falha em cascata](https://en.wikipedia.org/wiki/Cascading_failure). Traduzido da Wikipedia:
+
+> É um processo em um sistema de partes interconectadas em que a falha de uma ou poucas partes pode desencadear a falha de outras partes e assim por diante.
+
+Como aceitar falhas?
+
+ * Tolerância ao erro
+ * Degradação elegante (`Graceful degradation)
```
Comment (naXa777) [inline readme-translations/README.br.md]: ` - possible redundant acute (back quote)?

File: readme-translations/README.br.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+
+### Preparação
+
+Antes do primeiro build será necessário executar os seguintes passos adicionais: 
+
+ 1. Clonar o repositório
+ 2. Definir variáveis de ambiente
+ 3. &#x1F4D7; _TODO_
+
+Você tem liberdade para escolher sua ferramenta de build: [Gradle](https://gradle.org/) or your favourite IDE.
```
Comment (naXa777) [inline readme-translations/README.br.md]: This line is half-translated

File: readme-translations/README.br.md [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+[IntelliJ IDEA](https://spring.io/guides/gs/intellij-idea/), [STS](https://stackoverflow.com/q/34214685/1429387) / [Eclipse](http://www.vogella.com/tutorials/EclipseGradle/article.html), or [NetBeans](https://netbeans.org/features/java/build-tools.html) should handle this task smoothly.
```
Comment (naXa777) [inline readme-translations/README.br.md]: This line is not translated

**Review by naXa777 — APPROVED** [review]

**Comment by myakiju** [comment]
- [X] added crosslinks to PT-BR translation for other READMEs
- [X] translated missing parts

**Pull request merged**